package.json

```diff
@@ -24,11 +24,9 @@
     "haste": {
       "defaultPlatform": "ios",
       "providesModuleNodeModules": [
-        "fbjs",
         "react",
         "react-native",
-        "parse",
-        "react-transform-hmr"
+        "parse"
```

Comment:
This should use the exact same format that packager is using. I just asked @skevy to make it backwards compatible so that his change was non-breaking :)